tests/parser/syntax/test_block.py

```diff
@@ -126,9 +130,11 @@ def foo() -> uint256:
     return as_unitless_number(block.timestamp)
     """,
     """
+struct X:
+    x: timestamp
 @public
 def add_record():
-    a: {x: timestamp} = {x: block.timestamp}
+    a: X = X({x: block.timestamp})
```

Comment:
Add a failing test case for `X(<!= ast.Dict >)`.

Reply:
Fixed in 343dcfb0afb06403e9d3a04b23a6e5912463892d